CORE/step_definitions/page_steps.rb

```diff
@@ -11,12 +11,26 @@
   set_data_target
 end
 
+Given /^I am on the (.*?) Page(?: by way of the (.*?) Page)? by any means$/ do |target_page, intermediate_page|
```

Comment:
I see what you are going for, but the verbiage seems clunky.

Reply:
I'm a little curious if this should be the default behavior and maybe we have a flag that makes it use the old "Break if i ended up somewhere random" logic.

Reply:
I agree. I feel like this probably should become the default behavior, and some sort of flag or extended verbiage to produce the old functionality.

I like it though. 👍

Reply:
Am alternative, since i made core oz only lead its step defs when you explicitly cit tell it to in your setup file, is to just charge the default step defs to use proceed_to! And allow users to define custom steps that directly call proceed_to if they need it